.circleci/config.yml.in

```diff
@@ -81,6 +81,93 @@ jobs:
           paths:
             - "*"
 
+  binary_linux_conda_cuda:
+    <<: *binary_common
+    machine:
+      image: ubuntu-1604:201903-01
+    resource_class: gpu.medium
+    steps:
+    - checkout
+    - run:
+        name: Build
+        command: |
+          set -e
+
+          curl -L https://packagecloud.io/circleci/trusty/gpgkey | sudo apt-key add -
+          curl -L https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+
+          sudo apt-get update
+
+          sudo apt-get install \
+              apt-transport-https \
+              ca-certificates \
+              curl \
+              gnupg-agent \
+              software-properties-common
+
+          curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+          sudo add-apt-repository \
+             "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+             $(lsb_release -cs) \
+             stable"
+
+          sudo apt-get update
+          sudo apt-get install docker-ce docker-ce-cli containerd.io
+
+          # Add the package repositories
+          distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
+          curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | sudo apt-key add -
+          curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.list | sudo tee /etc/apt/sources.list.d/nvidia-docker.list
+
+          sudo apt-get update && sudo apt-get install -y nvidia-container-toolkit
+          sudo systemctl restart docker
+
+
+          DRIVER_FN="NVIDIA-Linux-x86_64-410.104.run"
+          wget "https://s3.amazonaws.com/ossci-linux/nvidia_driver/$DRIVER_FN"
+          sudo /bin/bash "$DRIVER_FN" -s --no-drm || (sudo cat /var/log/nvidia-installer.log && false)
+          nvidia-smi
+
+          # export DOCKER_IMAGE=nvidia/cuda:10.0-base
+          export DOCKER_IMAGE=soumith/conda-cuda
+          docker pull $DOCKER_IMAGE >/dev/null
+
+          export id=$(docker run --gpus all -t -d ${DOCKER_IMAGE})
+
+          docker cp /home/circleci/project/. $id:/workspace
+
+          cat >/home/circleci/install_and_test.sh \<<EOL
+          #!/bin/bash
+          set -e
+
+          curl -o conda.sh https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+          sh conda.sh -b
```

Comment:
If you're going to use `soumith/conda-cuda` image (which I'm not sure I condone, but I guess it's fine for now), you shouldn't need to reinstall conda; conda should already come on the image.

Reply:
This is a very good point. I originally tried using a basic `nvidia/cuda:10.0` image because I wanted kind of minimal dependencies, `conda-cuda` was huge (20GB) and I wanted to understand a bit a few more things.
But it was in the end a bit more work and I just switched to `soumith/conda-cuda`, which I have now found where the [original Dockerfile lives](https://github.com/pytorch/builder/blob/master/conda/Dockerfile), so I will switch to using it and use the conda from there as well.